Src/DependencyCollector/Shared.Tests/Implementation/ProfilerHttpProcessingTest.cs

```diff
@@ -115,6 +115,27 @@ public void RddTestHttpProcessingProfilerOnExceptionForGetResponse()
         }
 
         /// <summary>
+        /// Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.
+        /// </summary>
+        [TestMethod]
+        [Description("Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.")]
+        [Owner("cithomas")]
+        [TestCategory("CVT")]
+        public void RddTestHttpProcessingProfilerOnWebExceptionForGetResponse()
+        {
+            var request = WebRequest.Create(this.testUrl);
+            this.httpProcessingProfiler.OnBeginForGetResponse(request);
+            Thread.Sleep(this.sleepTimeMsecBetweenBeginAndEnd);
```

Comment:
These are old tests. I do not know how they work. You try to remove and see where it fails.

---
In reply to: [66872458](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/94#discussion_r66872458) [](ancestors = 66872458)